srcpkgs/superlu/template

```diff
@@ -1,18 +1,17 @@
 # Template file for 'superlu'
 pkgname=superlu
-version=5.2.1
+version=5.2.2
 revision=1
 build_style=cmake
+configure_args="-DUSE_XSDK_DEFAULTS=TRUE"
 hostmakedepends="gcc-fortran"
 makedepends="openblas-devel"
 short_desc="Solution of large, sparse, nonsymmetric systems of linear equations"
 maintainer="Julien Dehos <dehos@lisic.univ-littoral.fr>"
-license="BSD"
-homepage="http://crd-legacy.lbl.gov/~xiaoye/SuperLU/"
-distfiles="http://crd-legacy.lbl.gov/~xiaoye/SuperLU/${pkgname}_${version}.tar.gz"
-checksum=28fb66d6107ee66248d5cf508c79de03d0621852a0ddeba7301801d3d859f463
-configure_args="-DUSE_XSDK_DEFAULTS=TRUE"
-wrksrc="SuperLU_${version}"
+license="BSD-3-Clause"
+homepage="https://portal.nersc.gov/project/sparse/superlu/"
```

Comment:
I seem unable to load the homepage. I can load portal.nersc.gov fine though.

Reply:
The trailing slash is critical.

Reply:
Damn